includes/admin/class-wc-rest-payments-orders-controller.php

```diff
@@ -275,10 +270,6 @@ public function capture_authorization( WP_REST_Request $request ) {
 				);
 			}
 
-			// Actualize order status.
-			$charge = $intent->get_charge();
-			$this->order_service->mark_payment_capture_completed( $order, $intent_id, $result['status'], $charge->get_id() );
-
```

Comment:
This code is redundant, as `update_order_status_from_intent` updates the order status.